src/test/java/fruitshop/model/FruitTransactionTest.java

```diff
@@ -0,0 +1,15 @@
+package fruitshop.model;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class FruitTransactionTest {
+    @Test
+    void fruitTransactionWithNegativeAmount_NotOk() {
```

Comment:
```suggestion
    void of_NegativeAmount_NotOk() {
```
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/5cac84c5-f72d-446a-a249-fd0faafb4733)